WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Onboarding Errors/InPersonPaymentsCashOnDeliveryPaymentGatewayNotSetUpViewModel.swift

```diff
@@ -169,3 +175,14 @@ private enum Localization {
 private enum Constants {
     static let cashOnDeliveryGatewayID = "cod"
 }
+
+private extension CardPresentPaymentsPlugin {
+    var cashOnDeliveryLearnMoreURL: URL {
+        switch self {
+        case .wcPay:
+            return WooConstants.URLs.wcPayCashOnDeliveryLearnMoreUrl.asURL()
+        case .stripe:
+            return WooConstants.URLs.stripeCashOnDeliveryLearnMoreUrl.asURL()
```

Comment:
Should we remove the Url suffix from these 2 constants to avoid assumptions that are already an URL, as we need to convert them later: `somethingSomethingUrl.asURL()` 

I believe there's no general preference as we seem to use that suffix in some constants while we don't in others

Reply:
Yeah, I think that would be good since they're namespaces to `URLs` anyway.

I'll do it at some point when I'm done with iteration 2, and make them consistent.